ccdproc/ccddata.py

```diff
@@ -20,49 +20,60 @@
 
 
 class CCDData(NDDataArray):
-
-    """A class describing basic CCD data
+    """A class describing basic CCD data.
 
     The CCDData class is based on the NDData object and includes a data array,
     uncertainty frame, mask frame, meta data, units, and WCS information for a
     single CCD image.
 
     Parameters
     -----------
-    data : `~numpy.ndarray` or :class:`~ccdproc.CCDData`
+    data : `numpy.ndarray` or :class:`~ccdproc.CCDData`
         The actual data contained in this `~ccdproc.CCDData` object.
         Note that this will always be copies by *reference* , so you should
-        make copy the ``data`` before passing it in if that's the  desired
+        make copy the ``data`` before passing it in if that's the desired
         behavior.
 
-    uncertainty : `~astropy.nddata.StdDevUncertainty` or `~numpy.ndarray`,
-        optional Uncertainties on the data.
+    uncertainty : `~astropy.nddata.StdDevUncertainty` or `numpy.ndarray` or \
+            None, optional
+        Uncertainties on the data.
+        Default is ``None``.
```

Comment:
@bsipocz I didn't want to link to python-builtins constants like `None`, `False` or `True` or types `int`, `float`, `dict`, ...